src/issue/screens/new-issue.screen.js

```diff
@@ -90,17 +92,19 @@ class NewIssue extends Component {
       ).then(issue => {
         navigation.navigate('Issue', {
           issue,
+          headerLeft: false,
```

Comment:
Ah that error message is most likely because its only on the top Stack (see [here](https://github.com/gitpoint/git-point/blob/master/routes.js#L339-L366)) hence why it can't recognize the `Issue` screen :thinking: 

Glad to hear that just using `.reset` worked (somewhat) :P. The bottom menu item? Are you referring to the tab navigator?